delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/indexing/IndexingState.scala

```diff
@@ -0,0 +1,10 @@
+package ch.epfl.bluebrain.nexus.delta.sdk.indexing
+
+import ch.epfl.bluebrain.nexus.sourcing.projections.stream.StreamSupervisor
+
+sealed private[indexing] trait IndexingState extends Product with Serializable
+
+private[indexing] object IndexingState {
+  final case object Initial                                         extends IndexingState
```

Comment:
We could have a different state for Stop to make a distinction between views that have never been started and the ones that ran at least once

Reply:
Again, this information is going to be obtained using the progress